Bottom: b544e703bf229eb499608901a003e494390166b3
Top:    4bc0a9bfb43dd798a63029aaf559da8e5422381a
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:37:02 -0500

gitignore additions


---

diff --git a/.gitignore b/.gitignore
index d9d3967a12c..dd80578383e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+NOTES
 *.diff
 *.patch
 *.orig
